diesel/Cargo.toml

```diff
@@ -17,7 +17,7 @@ byteorder = "1.0"
 chrono = { version = "0.4.1", optional = true }
 libc = { version = "0.2.0", optional = true }
 libsqlite3-sys = { version = ">=0.8.0, <0.19.0", optional = true, features = ["min_sqlite_version_3_7_16"] }
-mysqlclient-sys = { version = "0.2.0", optional = true }
+mysqlclient-sys = { version = "0.2.4", optional = true }
```

Comment:
`mysqlclient-sys 0.2.2` is the first version that supports the necessary functions, therefore I do not see any reason to bump the version here to 0.2.4.

Reply:
We definitively need to update the minimal supported mysqlclient-sys version here to something newer than even released. My old comment was written with the assumption that the old release also contains all necessary functions. 

Generally speaking: We try not to update the minimal supported mysqlclient-sys version as that is considered a breaking change, due to the fact that it links a native library and cargo only allows one version of `mysqlclient-sys` in your dependency tree.

Reply:
04c4dc482d8cf3291213a74f810b9030078dd192 brings 0.2.2